src/vs/workbench/contrib/positronPlots/browser/webviewPlotClient.ts

```diff
@@ -63,13 +62,55 @@ export class WebviewPlotClient extends Disposable implements IPositronPlotClient
 		return undefined;
 	}
 
+	/** Whether the plot's underlying webview is active. */
+	isActive(): boolean {
+		return !!this._webview.value;
+	}
```

Comment:
If you turn this into a predicate function then you can reuse the narrowing later:
```suggestion
	isActive(): this is { _webview: { value: IOverlayWebview } } {
		return Boolean(this._webview.value);
	}
```

Then instead of 
```ts
if (!this._webview.value) {
...
}
```

You could do

```ts
if (!this.isActive()) {
...
}
```

Also I tend to find `Boolean(foo)` safer than `!!foo`.

Reply:
Oh neat, I didn't know about `this is`!